spring-cloud-sleuth-core/src/test/java/org/springframework/cloud/sleuth/documentation/SpringCloudSleuthDocTests.java

```diff
@@ -61,8 +63,8 @@ public Sampler defaultSampler() {
 	}
 
 	// tag::span_name_annotation[]
-	@SpanName("calculateTax")
-	class TaxCountingRunnable implements Runnable {
+	@SpanName("processTax")
```

Comment:
supernit: was this rename intentional? I think calculateTax is a decent name and also doesn't incidentally join the word Process with Runnable